Dockerfile

```diff
@@ -1,35 +1,21 @@
-FROM postgres:10.4-alpine
+FROM timescale/timescaledb:0.12.1-pg10
```

Comment:
Should we consider making this a `--build-arg` so that we can easily provide multiple `pg_prometheus` variants?

Reply:
Done